base/src/main/java/io/spine/protobuf/MessageFieldException.java

```diff
@@ -0,0 +1,56 @@
+package io.spine.protobuf;
+
+import com.google.protobuf.Message;
+
+import javax.annotation.Nullable;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+/**
+ * Signals an error working with a protobuf message field.
+ *
+ * @author Alexander Yevsyukov
+ */
+public class MessageFieldException extends RuntimeException {
+
+    private final Message protobufMessage;
+
+    /**
+     * Constructs a {@code MessageFieldException} with the formatted message text.
+     *
+     * @param protobufMessage    a Protobuf message object working with a field of which caused
+     *                           an error
+     * @param errorMessageFormat a formatted string for error message
```

Comment:
I would not call that a `formatted string`. That's a string template used for error message formatting.